pyp2rpm/virtualenv.py

```diff
@@ -42,7 +42,7 @@ def fill(self, path):
         '''
         self.bindir = set(os.listdir(path + 'bin/'))
         self.lib_sitepackages = set(os.listdir(glob.glob(
-            path + 'lib/python?.?/site-packages/')[0]))
+            path + 'lib/python*.*/site-packages/')[0]))
```

Comment:
We should probably use a proper regular expression here ... `*.*` is bound to blow up somehow.

Reply:
That string is an arg to the glob.glob() call.  We can't use regex there.  :)

This isn't strictly related to this branch.  There used to be more references to ?.?, which will break if Python 3.10 is the version after 3.9.  I missed this one in an earlier cleanup.

Reply:
I know that this is a glob.glob() call ... but it's still ugly :)
If it's solved like this everywhere else as well, then I'm fine with it.